src/main/kotlin/lotto/domain/Ranks.kt

```diff
@@ -0,0 +1,17 @@
+package lotto.domain
+
+data class Ranks(val ranks: List<Rank> = listOf()) {
+
+    val size = ranks.size
+
+    fun missCount(): Int = ranks.count { it == Rank.MISS }
+    fun fifthCount(): Int = ranks.count { it == Rank.FIFTH }
+    fun fourthCount(): Int = ranks.count { it == Rank.FOURTH }
+    fun thirdCount(): Int = ranks.count { it == Rank.THIRD }
+    fun secondCount(): Int = ranks.count { it == Rank.SECOND }
+    fun firstCount(): Int = ranks.count { it == Rank.FIRST }
```

Comment:
```suggestion
    fun rankCount(rank: Rank): Int = ranks.count { it == rank }
```
다른 방법으로는 위와 같은 함수를 정의하여 비슷한 기능을 하는 코드를 줄여볼수 있을것 같네요 😄